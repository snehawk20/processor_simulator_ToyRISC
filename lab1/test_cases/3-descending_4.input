	.data
a:
	80
	70
	60
	50
	90
	100
	110
	120
n:
	8
