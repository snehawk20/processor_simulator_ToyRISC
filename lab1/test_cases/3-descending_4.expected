Main Memory Contents:

0		: 120
1		: 110
2		: 100
3		: 90
4		: 80
5		: 70
6		: 60
7		: 50
